AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/SmokeTestsPackage.Base.txt

```diff
@@ -0,0 +1,39 @@
+// MARK: - Static Content
```

Comment:
The static content of package manifest. The only generated content, which comes right before this, is `serviceNames` array which will contain names of all services that have smoke tests. Then, that array is used in this static content for adding all the executable targets and executable products needed to build & run our test runners.